scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -36,6 +37,39 @@
 (meters/defmeter [cook-mesos scheduler handle-status-update-rate])
 
 
+(defn conditionally-sync-sandbox
+  [conn task-id task-state sync-agent-sandboxes-fn]
+  (let [instance-ent (d/entity (d/db conn) [:instance/task-id task-id])]
+    (when (and (#{:task-starting :task-running} task-state)
+               (not= :executor/cook (:instance/executor instance-ent)))
+      ;; cook executor tasks should automatically get sandbox directory updates
```

Comment:
Is there a cook executor for kubernetes??? This comment seems off.

Reply:
This is for mesos.